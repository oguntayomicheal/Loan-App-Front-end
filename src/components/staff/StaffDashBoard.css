#staffDashBoard {
    width: 80%;
    margin: auto;
}

#allLoans {
    padding: 0;
}

.eachLoan {
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 0 1%;
    border-radius: 8px;
}

.eachLoan:hover {
    border: 1px solid skyblue;
}

li {
    list-style: none;
}
