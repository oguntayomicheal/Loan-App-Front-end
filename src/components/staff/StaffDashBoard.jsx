import React from "react";

const StaffDashBoard = () => {


    return (
        <>
        Staff dashboard
        </>
    )
} 


export default StaffDashBoard;