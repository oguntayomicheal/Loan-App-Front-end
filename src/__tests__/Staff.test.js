import {render} from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'
import StaffDashBoard from '../components/staff/StaffDashBoard'
import ReviewLoanApplication from '../components/staff/ReviewLoanApplication'

describe('Staff page', () => {
    beforeEach(() => {
        localStorage.clear()
       
    })

    test('Renders staffDashBoard correctly', () => {
      
        localStorage.setItem('staffData', JSON.stringify({ id: 1 }));

        const staffDashBoard = render(
            <Router>
                <StaffDashBoard />
            </Router>
        )
        expect(staffDashBoard).toMatchSnapshot()
    })

    test('Renders staff loan review page correctly', () => {
      
        localStorage.setItem('staffData', JSON.stringify({ id: 1 }));

        const reviewLoanApplication = render(
            <Router>
                <ReviewLoanApplication />
            </Router>
        )
        expect(reviewLoanApplication).toMatchSnapshot()
    })


})