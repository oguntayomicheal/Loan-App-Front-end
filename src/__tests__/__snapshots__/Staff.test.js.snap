// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer loan status page Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff  status page Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff loan status page Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders dascorrectly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders review page correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders staff loan review page correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="main_content staff_page"
      >
        <div>
          <a
            href="/StaffDashBoard"
          >
            Back
          </a>
          <div
            class="loan_details"
          >
            <article>
              <h2>
                Loan Details
              </h2>
              <p>
                Customer name:
              </p>
              <p>
                Loan amount:
              </p>
              <p>
                Application Date:
              </p>
              <p>
                Purpose:
              </p>
              <p>
                Repayment preference:
              </p>
            </article>
            <article>
              <h2>
                Current review details
              </h2>
              <p>
                Last review date:
              </p>
              <p>
                Status:
              </p>
              <p>
                Details:
              </p>
              <pa>
                Amount to pay back:
              </pa>
              <p>
                Repayment Schedule:
              </p>
            </article>
          </div>
          <h2>
            Loan Application Review Form
          </h2>
          <form>
            <div
              class="form-group"
            >
              <label
                for="status"
              >
                Loan Status
                <br />
                <select
                  id="status"
                  name="status"
                  placeholder="Enter desired status"
                  required=""
                  type="text"
                >
                  <option
                    value=""
                  >
                    Select Status
                  </option>
                  <option
                    value="Approved"
                  >
                    Approved
                  </option>
                  <option
                    value="Rejected"
                  >
                    Rejected
                  </option>
                  <option
                    value="Closed"
                  >
                    Closed
                  </option>
                  <option
                    value="Pending"
                  >
                    Pending
                  </option>
                </select>
              </label>
            </div>
            <div
              class="form-group"
            >
              <label
                for="details"
              >
                <p>
                  Rejected ?, provide reasons. Approved?, Enter loan terms and conditions
                </p>
                <br />
                <input
                  id="details"
                  name="details"
                  placeholder="Enter further Details"
                  required=""
                  type="textarea"
                  value=""
                />
              </label>
            </div>
            <div
              class="form-group"
            >
              <label
                for="amount_to_pay"
              >
                Amount to pay
                <br />
                <input
                  id="amount_to_pay"
                  name="amount_to_pay"
                  placeholder="Enter the Amount to pay back"
                  type="number"
                  value=""
                />
              </label>
            </div>
            <div
              class="form-group"
            >
              <label
                for="repayment_schedule"
              >
                Repayment Schedule
                <br />
                <input
                  id="repayment_schedule"
                  name="repayment_schedule"
                  type="date"
                  value=""
                />
              </label>
            </div>
            <button
              type="submit"
            >
              Submit Review
            </button>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="main_content staff_page"
    >
      <div>
        <a
          href="/StaffDashBoard"
        >
          Back
        </a>
        <div
          class="loan_details"
        >
          <article>
            <h2>
              Loan Details
            </h2>
            <p>
              Customer name:
            </p>
            <p>
              Loan amount:
            </p>
            <p>
              Application Date:
            </p>
            <p>
              Purpose:
            </p>
            <p>
              Repayment preference:
            </p>
          </article>
          <article>
            <h2>
              Current review details
            </h2>
            <p>
              Last review date:
            </p>
            <p>
              Status:
            </p>
            <p>
              Details:
            </p>
            <pa>
              Amount to pay back:
            </pa>
            <p>
              Repayment Schedule:
            </p>
          </article>
        </div>
        <h2>
          Loan Application Review Form
        </h2>
        <form>
          <div
            class="form-group"
          >
            <label
              for="status"
            >
              Loan Status
              <br />
              <select
                id="status"
                name="status"
                placeholder="Enter desired status"
                required=""
                type="text"
              >
                <option
                  value=""
                >
                  Select Status
                </option>
                <option
                  value="Approved"
                >
                  Approved
                </option>
                <option
                  value="Rejected"
                >
                  Rejected
                </option>
                <option
                  value="Closed"
                >
                  Closed
                </option>
                <option
                  value="Pending"
                >
                  Pending
                </option>
              </select>
            </label>
          </div>
          <div
            class="form-group"
          >
            <label
              for="details"
            >
              <p>
                Rejected ?, provide reasons. Approved?, Enter loan terms and conditions
              </p>
              <br />
              <input
                id="details"
                name="details"
                placeholder="Enter further Details"
                required=""
                type="textarea"
                value=""
              />
            </label>
          </div>
          <div
            class="form-group"
          >
            <label
              for="amount_to_pay"
            >
              Amount to pay
              <br />
              <input
                id="amount_to_pay"
                name="amount_to_pay"
                placeholder="Enter the Amount to pay back"
                type="number"
                value=""
              />
            </label>
          </div>
          <div
            class="form-group"
          >
            <label
              for="repayment_schedule"
            >
              Repayment Schedule
              <br />
              <input
                id="repayment_schedule"
                name="repayment_schedule"
                type="date"
                value=""
              />
            </label>
          </div>
          <button
            type="submit"
          >
            Submit Review
          </button>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders staff review page correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders staffDashBoard correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders staffdashBoard correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Staff page Renders staffdashcorrectly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="staffDashBoard"
      >
        Staff dashboard
        <button
          class="logout_btn"
          type="button"
        >
          Logout
        </button>
        <h2>
          Loan Applications
        </h2>
        <ul
          id="allLoans"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="staffDashBoard"
    >
      Staff dashboard
      <button
        class="logout_btn"
        type="button"
      >
        Logout
      </button>
      <h2>
        Loan Applications
      </h2>
      <ul
        id="allLoans"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
